### PR TITLE
[PL-51473] PL 1.39.x Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-51473](https://harness.atlassian.net/browse/PL-51473)

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

[PL-51473]: https://harness.atlassian.net/browse/PL-51473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -79,6 +79,14 @@ The following deprecated API endpoints are longer supported:
 
 ## May 2024
 
+### Version 1.39.x<!--  May 27, 2024 -->
+
+#### Fixed issues
+
+- Keyboard navigation was not functioning for the User Select dropdown in the UserGroup form, and uncontrolled form submission occurred when pressing Enter. Updated the User Select dropdown to support keyboard navigation. Additionally, the form now properly handles uncontrolled submissions when the Enter key is pressed, allowing users to select items using the keyboard without unintended form submissions. (PL-51168, ZD-62169)
```
Comment (brian-f-harness) [inline release-notes/platform.md:86]: @saifeemustafaqharness I suggest watching the video in the Jira. The dropdown name in the UI is **Add Users**, not User Select. The issue occurred when adding new members to a user group.

Comment (brian-f-harness) [inline release-notes/platform.md:86]:
```suggestion
- Keyboard navigation was not functioning for the **Add Users** list when adding a new member to a user group, and uncontrolled form submission occurred when pressing Enter. Updated the **Add Users** list to support keyboard navigation. Additionally, the form now properly handles uncontrolled submissions when the Enter key is pressed, allowing users to select items using the keyboard without unintended form submissions. (PL-51168, ZD-62169)
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -79,6 +79,14 @@ The following deprecated API endpoints are longer supported:
...
+
+- Users could be added to SCIM Provisioned/Externally Managed User Groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and the addition of users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)
```
Comment (brian-f-harness) [inline release-notes/platform.md:88]:
```suggestion
- Users could be added to SCIM-provisioned/externally managed user groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and the addition of users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)
```

File: release-notes/platform.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Platform release notes
 sidebar_label: Platform
 tags: [NextGen, "platform"]
-date: 2024-05-20:T10:00:30
+date: 2024-05-27:T10:00:30
```
Comment (brian-f-harness) [inline release-notes/platform.md:5]:
```suggestion
date: 2024-05-28:T10:00:30
```

File: release-notes/platform.md [hunk 4]
```diff
@@ -79,6 +79,14 @@ The following deprecated API endpoints are longer supported:
 
 ## May 2024
 
+### Version 1.39.x<!--  May 27, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/platform.md:82]:
```suggestion
### Version 1.39.4<!--  May 27, 2024 -->
```

File: release-notes/platform.md [hunk 5]
```diff
@@ -79,7 +79,7 @@ The following deprecated API endpoints are longer supported:
 
 ## May 2024
 
-### Version 1.39.x<!--  May 27, 2024 -->
+### Version 1.39.4<!--  May 27, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/platform.md:82]:
```suggestion
### Version 1.39.4<!--  May 28, 2024 -->
```

**Review by brian-f-harness — DISMISSED** [review]

File: release-notes/platform.md [hunk 6]
```diff
@@ -79,6 +79,14 @@ The following deprecated API endpoints are longer supported:
 
 ## May 2024
 
+### Version 1.39.4<!--  May 28, 2024 -->
+
+#### Fixed issues
+
+- Keyboard navigation was not functioning for the **Add Users** list when adding a new member to a user group, and uncontrolled form submission occurred when pressing Enter. Updated the **Add Users** list to support keyboard navigation. Additionally, the form now properly handles uncontrolled submissions when the Enter key is pressed, allowing users to select items using the keyboard without unintended form submissions. (PL-51168, ZD-62169)
+
+- Users could be added to SCIM-provisioned/externally managed user groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and the addition of users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)
```
Comment (brian-f-harness) [inline release-notes/platform.md:88]:
```suggestion
- Users could be added to SCIM-provisioned/externally managed user groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and adding users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)
```

**Review by brian-f-harness — APPROVED** [review]

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**